BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -41,6 +46,15 @@
   private List<NoteItem> noteList;
   private List<NoteItem> selectedNoteItems;
 
+  private BookDao bookDao;
+  private NoteDao noteDao;
+
+  private ExportBibTex exportBibTex;
+  private String fileName;
+  private final String folderName = "Download";
+  private final String fileTypeBibToast = ".bib";
```

Comment:
The folderName and the fileTypeBibToast always have the same values. Maybe you can create a export constants class for them (e.g. see LibraryKeys class) and also use them in your other export code.